pkg/apis/samplesoperator/v1alpha1/types.go

```diff
@@ -119,6 +128,15 @@ func (s *SamplesResource) ConditionUpdate(c *SamplesResourceCondition) {
 	}
 }
 
+func (s *SamplesResource) StatusUpdate(c SamplesResourceConditionType) {
+	if c == SamplesExist {
+		s.Status.InstallType = s.Spec.InstallType
+		for _, arch := range s.Spec.Architectures {
+			s.Status.Architectures = append(s.Status.Architectures, arch)
```

Comment:
should do this in a way that ensures we don't acrue dupes in the future

Reply:
Had a change ready right before lunch, but given the direction to move this out of status and use a config map, this will get reworked anyway.

Perhaps with config maps, we can use key/value (with well defined keys) vs. arrays, and can by nature prevent the array accrual concern.  I'll sort it out as I revisit the ConfigMap particulars.